cognitive_service_vision_model_customization_python_samples/models/dataset_models.py

```diff
@@ -26,21 +26,23 @@ def __init__(self, kind: AuthenticationKind = AuthenticationKind.MI, sas_token=N
 
     @property
     def params(self):
-        return {
-            'kind': self.kind.value,
-            'sasToken': self.sas_token
+        params = {
+            'kind': self.kind.value
         }
+        if self.kind == AuthenticationKind.SAS:
+            params['sasToken'] = self.sas_token
+        return params
 
     @staticmethod
     def from_response(json):
         if not json:
             return None
 
-        return Authentication(json['kind'], json.get('sas'))
+        return Authentication(json['kind'], json.get('sasToken'))
```

Comment:
According to API, `sasToken` is the right name for this property - `sas` is incorrect